z3c/dependencychecker/package.py

```diff
@@ -209,5 +214,7 @@ def set_declared_extras_dependencies(self):
     def analyze_package(self):
         top_folder = self.metadata.top_level
         for module_obj in MODULES:
+            logger.debug(module_obj)
             for source_file in module_obj.create_from_files(top_folder):
+                logger.debug('Search deps on file %s', source_file.path)
```

Comment:
"on file" => "in file"?
(And 'deps' => 'dependencies'?)

Reply:
Done